contrib/for-tests/netexec/netexec.go

```diff
@@ -191,11 +193,53 @@ func dialUDP(request string, remoteAddress *net.UDPAddr) (string, error) {
 
 func shellHandler(w http.ResponseWriter, r *http.Request) {
 	log.Println(r.FormValue("shellCommand"))
-	output, err := exec.Command(shellPath, "-c", r.FormValue("shellCommand")).CombinedOutput()
-	assertNoError(err)
```

Comment:
I'd like it to keep running so that the client gets a response but, then again, it doesn't break anything to not respond back to the client for a test like this.